### DISCUSSION TITLE
Adding Microsoft Tests to pipeline

### DISCUSSION THREAD

**Discussion by cegekaJG** [microsoft/AL-Go#447] [Q&A]
What is the easiest way to create a test suite (using Microsoft's tests included in the test toolkit) that will be run alongside my test app? Can I create a `bctsuite.json` file like the one included in the performance test app? If I use a config package to import a test suite, will it be added to the tests?

**Comment by freddydk** [comment]
Is the question around a test suite or a bcpt test suite?

**Reply by cegekaJG** [reply]
I'm asking about the test suite.

**Comment by cegekaJG** [comment]
I'd like to bump this thread since I'm still interested in including the standard tests in the CI/CD tests.

**Comment by freddydk** [comment]
I will create an item in the backlog to be able to include Microsofts tests in the testsuite (using a setting or like)
